### ISSUE TITLE
Support additional properties in add/update

### ISSUE THREAD

**Issue by dshanske** [indieweb/wordpress-micropub#292]
https://github.com/indieweb/wordpress-micropub/blob/trunk/includes/class-micropub-endpoint.php#L520

@chrisbergr reported  https://github.com/dshanske/simple-location/issues/271 but this is actually the fact that the Micropub endpoint only supports two categories for adding or updating properties.

I can't remember why @snarfed and I left it like that, but it seems that we have a use case to fix the limitation.

**Comment by chrisbergr** [comment]
@dshanske To understand this correctly: the plugin only allows updates of the values "category" and "syndication" of a post, but because in this case a "photo" is passed, it fails?

If that's the case, I'm surprised I'm bringing the first usecase here :)

**Comment by dshanske** [comment]
Yes, it doesn't do it. I completely forgot we never implemented it
